src/app/core/layout/main-layout/main-layout.component.html

```diff
@@ -52,7 +51,7 @@
   >
     <button mat-flat-button color="primary" (click)="signOut()">
       <mat-icon>logout</mat-icon>
-      Sign Out ({{ signedInUser.name }})
+      Sign Out ({{ "signedInUser.name" }})
```

Comment:
You want to have "signedInUser.name" string rendered verbatim in the button instead of a user name, right? It can be simplified to:
`Sign Out (signedInUser.name)`

Reply:
yeah, anything that looks bad :D